viewing_party/main.py

```diff
@@ -0,0 +1,176 @@
+#Wave 1 
+
+def create_movie(movie_title, genre, rating):
+    new_movie = None
+    if movie_title and genre and rating:
+        new_movie = {
+            "title":movie_title,        "genre": genre,
+            "rating": rating 
+        }
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    watchlist = user_data["watchlist"] 
+    for movie in watchlist:
+        for movie_info in movie.values():
+            if movie_info == title:
+                user_data["watched"].append(movie)
+                watchlist.remove(movie)
+    return user_data
+
+
+#Wave 2
+
+def get_watched_avg_rating(user_data):
+    sum_rating = 0.0
+    watched_list = user_data["watched"]
+    for movie in watched_list:
+        for movie_val in movie.values():
+            if type(movie_val) == float:
```

Comment:
I'm curious why this line was added. It seems like it might be more appropriate for the function to add numbers even if they aren't float type.